packages/ui/Slider/index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import * as SliderPrimitive from "@radix-ui/react-slider";
+
+import { cn } from "@formbricks/lib/cn";
+
+interface SliderProps {
+  className?: string;
+  value: number;
+  max: number;
+  freeTierLimit: number;
```

Comment:
we are here in a generic Slider component. `freeTierLimit` is too specific for this. Please find a generic way for this slider or rename it to `BillingSlider`